src/ruby/install.sh

```diff
@@ -273,8 +278,10 @@ if [ "${SKIP_RBENV_RBUILD}" != "true" ]; then
             ln -s /usr/local/share/ruby-build /home/${USERNAME}/.rbenv/plugins/ruby-build
         fi
 
-        ln -s /usr/local/rvm/rubies/default/bin/ruby /usr/local/rvm/gems/default/bin 
-        
+        if [ ! -f /usr/local/rvm/gems/default/bin/ruby ]; then
+            ln -s /usr/local/rvm/rubies/default/bin/ruby /usr/local/rvm/gems/default/bin
```

Comment:
Do you know why we do this?

Reply:
Oryx expects ruby to be installed at  `/usr/local/rvm/gems/default/bin`, else it breaks the oryx flow of automatic installing gems. Hence, this bit was added. Added a comment!